app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people: list[dict]) -> list:
+    for person in people:
+        person_info = list(person.values())
+        Person(person_info[0], person_info[1])
```

Comment:
you can simply access the value of name and age by its key, there is no need to convert it to a list.